src/plugins/vis_type_vislib/public/fixtures/mock_data/histogram/_columns.js

```diff
@@ -25,7 +25,7 @@ export default {
       label: '404: response',
       xAxisLabel: 'machine.ram',
       ordered: {
-        interval: 100,
+        interval: 1000000,
```

Comment:
Why have we changed that?

Reply:
Those tests include a data range from 2 to 3 billion with an interval equal to 100 which seems to be not quite correct.
@nickofthyme [fix for vislib interval](https://github.com/DziyanaDzeraviankina/kibana/pull/1/files) includes `range`  function that generates a huge array with that data range and a small interval, which can't be processed, so tests fail to run. Another solution to that can be [calculating actual data interval instead of using the exactly given one](https://github.com/DziyanaDzeraviankina/kibana/commit/ef45b63c47da403399f76f00b49329531d445f31)

Reply:
@nickofthyme  I also think that [calculating actual data interval instead of using the exactly given one](https://github.com/DziyanaDzeraviankina/kibana/commit/ef45b63c47da403399f76f00b49329531d445f31) is a good idea 

We should visualize data based on ES response, not but Editor configuration.

Reply:
@alexwizp I don't exactly agree, if we allow setting the interval we should set it as close to the user-defined input as possible. The interval that is used in the ES request is the user-defined interval, with the expectation of the scaled interval limited by the `histogram:maxBars` setting. Vislib gets the data that conforms to the correct interval but renders with a different interval based on how the scales are created.

Maybe I am not understanding something here so I welcome any opinions.

I opened an issue to discuss this outside of this pr. https://github.com/elastic/kibana/issues/88693